annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/FastProcessor.java

```diff
@@ -80,25 +85,24 @@ private void reportDuplicates(String fastTypeName, Collection<Element> elements)
     private void buildCreatorClass(HashMap<String, Element> nameClassMap) throws IOException {
         String packageName = "com.exactpro.epfast.annotation.internal";
         String className = "CreatorImpl";
+        String interfaceName = "com.exactpro.epfast.annotation.ICreator";
+
+        HashMap<String, Object> scopes = new HashMap<>();
```

Comment:
Mustache can get value from POJO which is type-safe and preferable. So create class to hold these parameters